packages/chatbot-server-mongodb-public/evalCases/conversations.yml

```diff
@@ -1315,3 +1315,78 @@
     - atlas_cli
     - hallucination
     - user_feedback
+# Ops Manager / Cloud Manager FAQ
+- name: should understand OM/CM alerts
+  messages:
+    - content: Is there a way to generate an OpsManager alert for replica sets with an even number of nodes?
+      role: user
+  expectation: >
+    The ASSISTANT should say that Ops Manager does not provide a built-in alert condition specifically for detecting replica sets with an even number of voting members. However, the user can create a custom alert condition to monitor the number of voting members in a replica set.
+  tags:
+    - ops-manager
+    - cloud-manager
+    - alerts
+- name: should understand OM/CM user account settings
+  messages:
+    - content: Can I reset my password?
```

Comment:
this eval isn't really for om/cm. can also be atlas. maybe specify product in the query?